apps/dashboard/app/helpers/application_helper.rb

```diff
@@ -68,11 +68,11 @@ def app_icon_tag(app)
     end
   end
 
-  def icon_tag(icon_uri)
+  def icon_tag(icon_uri, classes: 'app-icon')
```

Comment:
```suggestion
  def icon_tag(icon_uri, classes: ['app-icon'])
```

Seems like this is an array in `image_tag` right?

Reply:
The helpers used to render the image (`fa_icon` and `image_tag`) support both, string and array.

But will update if it makes the interface clearer